packages/bpk-theming/src/BpkThemeProvider-test.js

```diff
@@ -95,6 +95,7 @@ describe('BpkThemeProvider', () => {
 
   it('should warn about missing theme attributes', () => {
     expect(
+      // eslint-disable-next-line react/forbid-foreign-prop-types
```

Comment:
why not doing the same thing here, i.e. exporting the propTypes?

Reply:
I have no idea maybe i got lazy 😂 

Will fix.